HPC/Gromacs.md

```diff
@@ -0,0 +1,122 @@
+Gromacs is one of the most widely used molecular dynamics software packages. Below is the instructions to build and run Gromacs on Hpc7g instances.
+
+## Build Gromacs
+* build cmake
```

Comment:
cmake 3.26 is required? Ubuntu-22.04 comes with cmake 3.22.1 (apt install...).

Reply:
gromacs-2022.4 requires minimum cmake version 3.16.3; some ec2 images comes with older version and requires an update to cmake.